### DISCUSSION TITLE
Find and restart failed integrations

### DISCUSSION THREAD

**Discussion by Choleriker** [frenck/spook#816] [Ideas]
I'm searching for something similar. I want to restart failed integrations. That happens when once a day my router restarts, then some integrations are in state failed. That I want to fix with a script where I can find failed integrations and restart them.

**Comment by frenck** [comment]
Failed integration should automatically restart; if they don't, I would consider that a bug.

Your proposal is not a solution.

../Frenck

**Reply by Choleriker** [reply]
Thats not true! Home Assistant does not restart automatically failed integrations. From where do you have this information?

**Reply by frenck** [reply]
Uh that is true. 

> From where do you have this information?

From myself, you might not realize, but I'm one of the Home Assistant Core architects and developers...
